deploy.sh

```diff
@@ -41,8 +53,8 @@ docker run --rm -v $PWD:${CODEBASE_PATH} \
 # TODO: is there a smaller version of this image we can use?
 docker run --rm -v $PWD:${CODEBASE_PATH} \
     -e BASE_URL="https://us-central1-broad-dsde-${ENVIRONMENT}.cloudfunctions.net" \
-    google/cloud-sdk:latest /bin/bash -c \
+    google/cloud-sdk:220.0.0 /bin/bash -c \
```

Comment:
pinning the cloud-sdk version to proactively avoid any problems with breaking changes in the floating "latest" tag.